src/docs/user/references/Variables.adoc

```diff
@@ -16,14 +16,21 @@ only within the task execution context.
 [[_job_variables]]
 ===== Job variables
 
-In a workflow you can define *job variables* that are shared and visible by all tasks.
+In a workflow, you can define *job variables* that are shared and visible by all tasks.
 
 A job variable is defined using the following attributes:
 
-* *name*: the name of the variable
-* *value*: the value of the variable
-* *model*: the type of the value (optional). See section <<_variable_model,Variable Model>>
-
+* *name*: the name of the variable.
```

Comment:
the variable name,
the variable value 
the variable type or model
the variable description
the group name